recipes/afwizard/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "afwizard" %}
+{% set version = "1.0.0b4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/afwizard-{{ version }}.tar.gz
+  sha256: 7754f55c2ffd5ad6b27123d803db765491f3212de735b38fb6b6ebb437bb4f9b
+
+build:
+  entry_points:
+    - afwizard = afwizard.__main__:main
+    - extract_opals_schema = afwizard.opals:_automated_opals_schema
+    - upgrade_filter_library = afwizard.library:upgrade_filter_library
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.9
+  run:
+    - afwizard-library
+    - click
+    - gdal
+    - geojson
+    - ipyfilechooser
+    - ipyleaflet
+    - ipython
+    - ipywidgets ==8.0.0rc0
+    - ipywidgets-jsonschema
+    - jsonmerge
+    - jsonschema <4
+    - jupyter-resource-usage
+    - jupyterlab
+    - numpy
+    - pdal
+    - pillow
+    - pyproj
+    - pyrsistent
+    - python >=3.9
+    - python-pdal
+    - pytools
+    - wrapt
+    - xdg
+    - xmltodict
+
+test:
+  imports:
+    - afwizard
+  commands:
+    - afwizard --help
+    - extract_opals_schema --help
+    - upgrade_filter_library --help
+
+about:
+  home: https://pypi.org/project/afwizard/
+  summary: Adaptive Filtering Wizard
+  license: MIT
+  license_file: LICENSE.md
```

Comment:
```suggestion
  license_file: LICENSE.md
  dev_url: https://github.com/ssciwr/afwizard
  doc_url: https://afwizard.readthedocs.io/
```

Reply:
Added. I did not know about the existence of these additional URLs, very helpful.